tutorials/FITS-images/FITS-images.ipynb

```diff
@@ -114,56 +201,128 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "gp63I_0dgEIs"
+   },
    "source": [
-    "Great! At this point, we can close the FITS file because we've stored everything we wanted to a variable."
+    "Great! At this point, we can close the FITS file because we've stored everything we wanted to a variable. By closing the FITS file, we can make sure that it won't continue using up excess memory or file handles on the computer."
```

Comment:
Hopefully, yes...it turns out that the situation gets more complicated depending on whether the file is read into memory or not. Not worth mentioning that in the tutorial, but `close` is not as clean as it should be.

Reply:
Okay, thanks for bringing to our attention! We'll keep it the same for now.